Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -196,6 +141,31 @@ - (void)updateRowHeight {
     [self.tableView reloadData];
 }
 
+
+#pragma mark - Notifications
```

Comment:
what does "notifications" mean to Simplenote?

Reply:
In this context, it refers to [this](https://developer.apple.com/documentation/foundation/nsnotificationcenter).

**NSNotificationCenter** is a publisher / subscriber implementation, meant to channel events in-app. There are Notifications posted by the OS itself / UIKit components, and notes posted by ourselves.

The pragma mark in here reads as `NSNotification handling methods below`, so to speak!

Reply:
interesting. a kind of app message-bus?

Reply:
Yes!! sent you details about this over slack!